docs/content/Configuration/Environment-Variables-Reference.md

```diff
@@ -166,17 +164,18 @@ databases [in this guide][link-connecting-to-db].
 
 ### <--{"id" : "Cube Store"}--> Cloud Storage
 
-| Environment variable              | Description                                                                                             | Possible Values                                                                         |
-| --------------------------------- | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
-| `CUBESTORE_AWS_ACCESS_KEY_ID`     | The Access Key ID for AWS. Required when using AWS S3                                                   | [A valid AWS Access Key ID][link-aws-creds]                                             |
-| `CUBESTORE_AWS_SECRET_ACCESS_KEY` | The Secret Access Key for AWS. Required when using AWS S3                                               | [A valid AWS Secret Access Key][link-aws-creds]                                         |
-| `CUBESTORE_S3_BUCKET`             | The name of a bucket in AWS S3. Required when using AWS S3                                              | A valid bucket name in the AWS account                                                  |
-| `CUBESTORE_S3_REGION`             | The region of a bucket in AWS S3. Required when using AWS S3                                            | [A valid AWS region][link-aws-regions]                                                  |
-| `CUBESTORE_S3_SUB_PATH`           | The path in a AWS S3 bucket to store pre-aggregations. Optional                                         | -                                                                                       |
-| `CUBESTORE_GCP_CREDENTIALS`       | A Base64 encoded JSON key file for connecting to Google Cloud. Required when using Google Cloud Storage | [A valid Google BigQuery JSON key file encoded as a Base64 string][link-gcp-creds-json] |
-| `CUBESTORE_GCP_KEY_FILE`          | The path to a JSON key file for connecting to Google Cloud. Required when using Google Cloud Storage    | [A valid Google Cloud JSON key file][link-gcp-creds-json]                               |
-| `CUBESTORE_GCS_BUCKET`            | The name of a bucket in GCS. Required when using GCS                                                    | A valid bucket name in the Google Cloud account                                         |
-| `CUBESTORE_GCS_SUB_PATH`          | The path in a GCS bucket to store pre-aggregations. Optional                                            | -                                                                                       |
+| Environment variable                     | Description                                                                                                                             | Possible Values                                                                         |
```

Comment:
@hassankhan, would it make sense to create a separate column in the table to indicate whether an env. variable is required vs. option? Not sure if that's overkill. Feel free to ignore this 😄

Reply:
It was previously discussed and we decided against it because it would add too many columns to the data. Definitely need to revisit how this page looks in the future though